api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -224,6 +225,27 @@ message TLSProperties {
 
   // SNI hostname from handshake.
   string tls_sni_hostname = 3;
+
+  message CertificateProperties {
+    message SubjectAltName {
+      oneof san {
+        string uri = 1;
+        string dns = 2;
```

Comment:
Is this used?

Reply:
no, i'll remove

Reply:
A not implemented annotation is also fine.